bin/wiki_entity_linking/wikidata_pretrain_kb.py

```diff
@@ -79,64 +87,66 @@ def main(
         )
 
     # STEP 2: create prior probabilities from WP
-    print()
-    if loc_prior_prob:
-        print(now(), "STEP 2: reading prior probabilities from", loc_prior_prob)
-    else:
+    if not prior_prob_path.exists():
```

Comment:
What's the motivation to remove these additional logging prints? I found them informative when I was running the scripts themselves, as they make sure that all steps are in the logging output and you can review which parts were newly created (the "writing" statements) and which were used as input (the "reading" statements)

Reply:
I'll add them back in